packages/org.openapis.v3/Link.pkl

```diff
@@ -0,0 +1,68 @@
+//===----------------------------------------------------------------------===//
+// Copyright © 2024 Apple Inc. and the Pkl project authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//===----------------------------------------------------------------------===//
+/// The Link object represents a possible design-time link for a response.
+///
+/// The presence of a link does not guarantee the caller's ability to
+/// successfully invoke it, rather it provides a known relationship and 
+/// traversal mechanism between responses and other operations.
+///
+/// Unlike dynamic links (i.e. links provided in the response payload), the 
+/// OAS linking mechanism does not require link information in the runtime 
+/// response.
+///
+/// For computing links, and providing instructions to execute them, a 
+/// runtime expression is used for accessing values in an operation and using 
+/// them as parameters while invoking the linked operation.
+///
+/// A linked operation MUST be identified using either an operationRef or 
+/// operationId. In the case of an operationId, it MUST be unique and 
+/// resolved in the scope of the OAS document. Because of the potential 
+/// for name clashes, the operationRef syntax is preferred for specifications 
+/// with external references.
```

Comment:
```suggestion
/// The Link object represents a possible design-time link for a response.
///
/// The presence of a link does not guarantee the caller's ability to
/// successfully invoke it, rather it provides a known relationship and
/// traversal mechanism between responses and other operations.
///
/// Unlike dynamic links (i.e. links provided in the response payload), the
/// OAS linking mechanism does not require link information in the runtime
/// response.
///
/// For computing links, and providing instructions to execute them, a
/// runtime expression is used for accessing values in an operation and using
/// them as parameters while invoking the linked operation.
///
/// A linked operation MUST be identified using either an operationRef or
/// operationId. In the case of an operationId, it MUST be unique and
/// resolved in the scope of the OAS document. Because of the potential
/// for name clashes, the operationRef syntax is preferred for specifications
/// with external references.
```